core/plugins/org.fusesource.ide.camel.model.service.core/src/org/fusesource/ide/camel/model/service/core/catalog/CamelModelFactory.java

```diff
@@ -46,10 +49,12 @@
 		camelVersionToFuseBOMMapping.put("2.15.1.redhat-621117", "6.2.1.redhat-117");
 		camelVersionToFuseBOMMapping.put("2.17.0.redhat-630187", "6.3.0.redhat-187");
 		camelVersionToFuseBOMMapping.put("2.17.3",               "6.3.0.redhat-187");
-		camelVersionToFuseBOMMapping.put("2.18.1",               "6.3.0.redhat-187");
```

Comment:
we can't include it as there is a the blocking regression with Blueprint: no Blueprint Camel route are working with Community 2.18.1